main.js

```diff
@@ -33,25 +43,18 @@ app.disableHardwareAcceleration();
 // The PSP will have a single instance. If an attempt to start a second instance is made,
 // the second one will be closed and the callback provided to `app.makeSingleInstance`
 // in the first instance will be triggered enabling it to show the PSP `BrowserWindow`.
-var appIsRunning = app.makeSingleInstance(function (commandLine) {
+app.on("second-instance", function (event, commandLine) {
     var qssWrapper = fluid.queryIoCSelector(fluid.rootComponent, "gpii.app.qssWrapper")[0];
     qssWrapper.qss.show();
-
     if (commandLine.indexOf("--reset") > -1) {
```

Comment:
This is the main fix here. The above replacement of `makeSingleInstance` with `app.on("second-instance")` makes no difference to the problem, but it's deprecated in electron 3 (and removed in 4)